pom.xml

```diff
@@ -472,6 +474,21 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>

```

Comment:
I'd rather not include it, mainly because of two reasons
* You can still `mvn pmd:check` or `mvn pmd:pmd` and get the same result
* Having a quick look at the report, it suggests a lot of changes that (for me) make the code harder to read without any real benefit (collapsible ifs, superfluous parenthesis), so I'm not very comfortable blindly following the report

WDYT?